sign/dilithium/gen.go

```diff
@@ -228,3 +232,85 @@ func generateModePackageFiles() {
 		}
 	}
 }
+
+// Copies mode3 source files to other modes
```

Comment:
Why not to move everything to templates and generate from them the implementations.

Reply:
Because then it's easy to test changes by developing them in mode3 (without having to run go generate.)